### ISSUE TITLE
Cut a new release (v2.5.0.beta2) to rubygems?

### ISSUE THREAD

**Issue by jcarbo** [infinitered/ProMotion#764]
The current version (2.5.0.beta1) is unusable with RedPotion, which I'm assuming is a major use-case since RedPotion ships with ProMotion. Looks like a fix was merged in a while ago (https://github.com/infinitered/ProMotion/commit/ea49c2d14c9e84d4a0cfd65467323734f416fa26) and the current build is green. Any way we can get a new version out?

Also minor plug for a PR I just opened (https://github.com/infinitered/ProMotion/pull/763) to add section footer functionality. It's a backwards-compatible change so I'd love to get that reviewed and possibly make it in to the next release.

--

Two other questions for the maintainers (cc @jamonholmgren specifically).
- There are a lot of open issues, many of which have already been addressed. For example, issue https://github.com/infinitered/ProMotion/issues/617 (currently open) was fixed via PR https://github.com/infinitered/ProMotion/issues/622. I'd be happy to identify some of these just wondering the best way to do it. Comment on the issues?
- What are the plans for the future of this project? I saw some conversation here (https://github.com/infinitered/ProMotion/issues/631) but nothing recently. We're starting to use RubyMotion (specifically the RedPotion/ProMotion stack) at my company and we're loving it so far but I'd be interested to know what the overall direction is.

**Comment by jamonholmgren** [comment]
@jcarbo Thanks for bringing this up!

TL;DR version: Lack of time, still plan to support this, please help! Also, yes, new release tomorrow.

Longer version:

I recently merged my company (ClearSight) into InfiniteRed to create the new Infinite Red, Inc. My role has changed to COO, so my focus has been on operations, rather than engineering. I'm also in the middle of building a new house, so my free time is often devoted to that. Because of this, I haven't done nearly as much coding in a long time.

Since I was the main person handling releases and issues, those are the things that have suffered the most.

We still use ProMotion every day (and RedPotion). We're often using from `master`, but we realize that people are depending on these libraries, so I'll make an effort to get caught back up.

If you're able to help by triaging issues (just comment on them), that would be fantastic. It would help me focus my time.

I'll commit to getting a new version out tomorrow (November 20). At this point, I think ProMotion is stable enough to release 2.5.0. I'll see if @twerth can also release RMQ and a new RedPotion as well.

Thanks for the friendly bump!

Jamon

**Comment by jcarbo** [comment]
Sounds good :+1:

**Comment by jamonholmgren** [comment]
@jcarbo Feel free to jump on Motioneers Slack: http://motioneers.herokuapp.com/

**Comment by jcarbo** [comment]
@jamonholmgren - when you release this new version can you also bump the version of RedPotion and cut a new release? When running both from master they work fine but the current PM master breaks with the RP 1.4.0 on rubygems.

**Comment by bmichotte** [comment]
I'm using :head for ProMotion and RedPotion, and yes, they are really stable together

**Comment by jamonholmgren** [comment]
@jcarbo Yes, I'll coordinate with Todd on that.
